packages/zowe-explorer-api/src/profiles/ProfilesCache.ts

```diff
@@ -316,10 +316,17 @@ export class ProfilesCache {
     }
 
     // This will retrieve the base profile from imperative
-    public async fetchBaseProfile(): Promise<imperative.IProfileLoaded | undefined> {
+    public async fetchBaseProfile(profileName?: string): Promise<imperative.IProfileLoaded | undefined> {
         const mProfileInfo = await this.getProfileInfo();
         const baseProfileAttrs = mProfileInfo.getDefaultProfile("base");
-        if (baseProfileAttrs == null) {
+        const configApi = mProfileInfo.getTeamConfig().api;
+        if (
+            profileName?.includes(".") &&
```

Comment:
I guess this is an opt-in feature. Meaning that extenders, calling the fetchBaseProfile directly, will have to provide the serviceProfile name or use our `log[in|out]WithBaseProfile` method 😋 

How should we let them know about it?
Will some TSDoc be warranted here? 😋

Reply:
Added TSDoc to the method and called out the change in ZE API release notes to inform extenders.